### PR TITLE
Popup functionality

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

File: src/apps/Casa/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ const CasaExample = () => {
   });
 
   zoneLayer.onClick(f => {
-    alert(`Clicked on ${f.length} ZoneLayer features.`);
+      console.log('Clicked', f);
```
Comment (gevos) [inline src/apps/Casa/README.md]: Have you linted this Readme?

File: src/components/Popup/Popup.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.tm-popup {
+  .tm-popup-row {
```
Comment (gevos) [inline src/components/Popup/Popup.scss]: Is this a space holder?

File: src/apps/Casa/README.md (hunk 1 above)
Comment (mhaertwig) [reply src/apps/Casa/README.md]: This is for demo reasons and has to be there.

**Comment by mhaertwig** [comment]
There are open merge requests in `react-spatial` which is why e.g. the `VectorLayer` does not work.

File: src/apps/Casa/README.md (hunk 1 above)
Comment (gevos) [reply src/apps/Casa/README.md]: console.log() is okay for demo. I meant the spaces and all the other linting errors. I will accept them for the demo, too.

**Pull request merged**